### PR TITLE
Checkbox, RadioButton, Switch: Add usage guidelines

### PR DESCRIPTION
**Pull request by ayeshakmaz**
### Summary

Add usage guidelines for these related components

### Links

- [Jira](https://jira.pinadmin.com/browse/PDS-2951)
- [TDD](link to Paper doc)
- [Figma](link to Figma file)

### CODE REVIEW

File: docs/src/Switch.doc.js [hunk 1]
```diff
@@ -55,6 +56,30 @@ card(
...
+          - For a binary option that can be either active or inactive.
+          - Typically used on mobile, where toggling the Switch takes immediate effect.
+        `}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        title="When Not to Use"
+        description={`
+          - Choosing between multiple, related options. Use [Checkboxes](Checkbox) or [RadioButtons](/RadioButton) instead.
```
Comment (AlbertCarreras) [inline docs/src/Switch.doc.js:76]: This one is kinda confusing, i don't think it adds much and could led to confusion if you need +1 switches together for any reason.

**Review by AlbertCarreras — APPROVED** [review]
LGTM, with one tiny comment

File: docs/src/Switch.doc.js (hunk 1 above)
Comment (ayeshakmaz) [reply docs/src/Switch.doc.js]: what do you think of the updated one?

**Comment by AlbertCarreras** [comment]
LGTM!

**Pull request merged**